Jenkinsfile

```diff
@@ -12,7 +12,9 @@ pipeline {
             CONFIG_REPO_NAME = 'edx-mobile-config'
             TEST_PROJECT_REPO_NAME = 'edx-app-test'
             AUT_NAME = 'edx-debuggable-2.23.2.apk'
-    }       
+            login_user_name = credentials('AUTOMATION_USERNAME')
+            login_user_password = credentials('AUTOMATION_PASSWORD')
```

Comment:
better to user Upper case letters as parity with other environment variables.

Reply:
fixed